### ISSUE TITLE
Reuse downloaded terraform version between multiple actions

### ISSUE THREAD

**Issue by donovanmuller** [dflook/terraform-github-actions#30]
If you combo multiple actions together in a single job, the Terraform version is downloaded for every action. It would be beneficial to share the same Terraform version between actions. I.e. The first action downloads `terraform` if the binary doesn't exist and the same version is reused in subsequent actions.

E.g. workflow
```yaml
...
jobs:
  cluster:
    steps:
      - name: Checkout
        uses: actions/checkout@v2

      - name: Get branch name
        uses: nelonoel/branch-name@v1.0.1

      - name: Use branch workspace
        uses: dflook/terraform-new-workspace@v1
        env:
          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
        with:
          path: blah
          workspace: ${{ env.BRANCH_NAME }}

      - name: Create cluster
        uses: dflook/terraform-apply@v1
        env:
          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
        with:
          workspace: ${{ env.BRANCH_NAME }}
          path: blah
          auto_approve: true
```

results in two downloads:

```
Run dflook/terraform-new-workspace@v1
...

Reading latest terraform version
Downloading https://releases.hashicorp.com/terraform/0.14.4/terraform_0.14.4_linux_amd64.zip to terraform_0.14.4_linux_amd64.zip
Downloading ...
33518568 bytes downloaded.
Switched terraform to version "0.14.4" 

...

Run dflook/terraform-apply@v1
...

Reading latest terraform version
Downloading https://releases.hashicorp.com/terraform/0.14.4/terraform_0.14.4_linux_amd64.zip to terraform_0.14.4_linux_amd64.zip
Downloading ...
33518568 bytes downloaded.
Switched terraform to version "0.14.4" 
```

**Comment by dflook** [comment]
Hi @donovanmuller, thanks for creating an issue. This is a good idea

**Comment by dflook** [comment]
Release v1.5.2 will only download the terraform binary once per job, if required. The terraform version included in the image has also been updated to 0.14.4.
